_pages/faq.md

```diff
@@ -5,31 +5,64 @@ permalink: /faq/
 layout: page
 ---
 
-## How do I suggest new or different data sources?
-If you have feedback on the data sources we have used or have suggestions for new data sources then please email us at <a href="mailto:{{site.email_contacts.questions}}">{{site.email_contacts.questions}}</a>.
+## Table of contents
+- [How often will new data be added to this site?](#how-often-will-new-data-be-added-to-this-site)
 
-## What does the reporting status mean?
-We have used three different types of reporting status for an indicator, which are also colour coded:
+- [How do I suggest new or different data sources?](#how-do-i-suggest-new-or-different-data-sources)
 
-* Reported online (black) – as a minimum the headline national data for this indicator is available on this website but the data might not be fully disaggregated yet. We are continuing to source additional disaggregations.
-* Statistics in progress (grey) – we have found a suitable source of data for this indicator or relevant proxy at national level. We are currently quality assuring the data and preparing it for publication.
-* Exploring data sources (white) – we are still looking for a suitable data source for this indicator.
+- [What are you doing to fill data gaps?](#what-are-you-doing-to-fill-data-gaps)
 
-Where there is additional information about the status of indicator data collection and reporting, this will be displayed at the top of the indicator page.
+- [What does the reporting status mean?](#what-does-the-reporting-status-mean)
+
+- [Can other countries copy this website?](#can-other-countries-copy-this-website)
+
+- [How accessible is this website?](#how-accessible-is-this-website)
+
+- [What browsers can I use to view this website?](#what-browsers-can-i-use-to-view-this-website)
+
+- [What do each of the three tiers mean?](#what-do-each-of-the-three-tiers-mean)
 
 ## How often will new data be added to this site?
```

Comment:
@brockfanning @AnnCorp should we remove the "in cooperation with topic experts bit" - just thinking that may be misleading if countries aren't working with topic experts